src/injected/analyzers/base-analyzer.ts

```diff
@@ -2,17 +2,19 @@
 // Licensed under the MIT License.
 import { autobind } from '@uifabric/utilities';
 import * as Q from 'q';
+
 import { VisualizationType } from '../../common/types/visualization-type';
 import { Analyzer, AnalyzerConfiguration, AxeAnalyzerResult, ScanCompletedPayload } from './analyzer';
 
-export type MessageType = {
-    // tslint:disable-next-line: no-reserved-keywords
-    type: string;
-    payload: ScanCompletedPayload<any>;
+export type ScanCompletedMessageType = MessageType<ScanCompletedPayload<any>>;
+
+export type MessageType<Payload = {}> = {
+    messageType: string;
+    payload: Payload;
 };
 
 export class BaseAnalyzer implements Analyzer {
-    protected sendMessage: (message) => void;
+    protected sendMessage: (message: MessageType) => void;
```

Comment:
Not using `Message` type in here (and instead use a '2nd' type for the messages we send to the background) will not prevent this from happening again.